### DISCUSSION TITLE
This is not the right place for this bug, but most logitech projects seem defunct.

### DISCUSSION THREAD

**Discussion by rlewkowicz** [LMS-Community/slimserver#796] [General]
I don't know exactly what this is, but this is amazing and it's disappointing there's not more community and information surrounding it: https://github.com/Logitech/hackzurich2019

I'm just hoping anyone here could give me any more information on logi-devmon.exe (source code?) and all the available paths and verbs. 

This would be the most killer feature of any mouse on the planet. Just the infinite programable customization and the ability to track and (possibly, docs are ambiguous) modify events would be AMAZING. I don't understand how this didn't take off more and isn't some sort of official project. I would take this over junky half functional ui's and lua scripts. 

Is anyone able to get me more info here? I know there's logitech folks on this project.

**Comment by mherger** [comment]
Fortunately you have found the right place to post, too. I'm closing this issue here.

**Comment by rlewkowicz** [comment]
@mherger  yeah, but you know I won't hear anything. I wonder if all that died because of covid. I emailed the main guy, but I doubt I'd hear anything. Even if you have an internal slack, even some history on it would be cool. Idk, that project just strikes me as like, 2010 a sort of fun and looseness that I haven't seen in a while. It's this short single page blog style thing with clean well written examples in every language. This is the seed of something amazing, and I'd love to know why it didn't grow.
